sysbuild/Kconfig.netcore

```diff
@@ -145,6 +164,25 @@ config NETCORE_APP_UPDATE
 	  Enabling this will add support for network core updates from the application core,
 	  requires b0n for network core and MCUboot for application core.
 
-endif
+if NETCORE_IPC_RADIO
+
+config NETCORE_IPC_RADIO_BT_HCI_IPC
+	bool
+	help
+	  Use HCI serialization for Bluetooth
+
+config NETCORE_IPC_RADIO_BT_RPC
+	bool
+	help
+	  Use nRF RPC serialization for Bluetooth
+
+config NETCORE_IPC_RADIO_IEEE802154
+	bool
+	help
+	  Use Spinel serialization for IEEE 802.15.4
```

Comment:
Config, or choice? Should only one of these be selected or can multiple? Should one of these by selected by default? Should it throw an error if one is not selected (and if so, error should be thrown at compile time in .c file, not in cmake)

Reply:
This is a config, multiple choices can be selected. The only limitation is available memory

Reply:
Would throw an error in the .c file as an assert if none of them are selected

Reply:
I can add an n-select to Bluetooths because I don't think someone will use both for any reason (but it should be possible).

Reply:
There will be no error when none is selected. When none is selected the application would act similar to "empty_net_core" sample.

Reply:
> I can add an n-select to Bluetooths because I don't think someone will use both for any reason (but it should be possible).

If supported then can be left as-is

> There will be no error when none is selected. When none is selected the application would act similar to "empty_net_core" sample.

Wouldn't that point to an error in the user's configuration that they should be aware of?

Reply:
I tested again and it turns out that bt_rpc and hci have different conflicting configurations so they can't be used together. In the next push I correct the use.

Reply:
The application can work with bt_rpc + 802.15.4, but the memory is limited in this configurtion. There are no restrictions (code or dependencies) against it.

Reply:
Thanks for the clarification